android/quest/src/main/java/org/smartregister/fhircore/quest/ui/login/LoginViewModel.kt

```diff
@@ -156,7 +156,10 @@ constructor(
             _showProgressBar.postValue(false)
             _loginErrorState.postValue(LoginErrorState.INVALID_OFFLINE_STATE)
           } else if (
-            accountAuthenticator.validateLoginCredentials(trimmedUsername, passwordAsCharArray)
+            accountAuthenticator.validateLoginCredentials(
+              trimmedUsername,
+              passwordAsCharArray,
+            )
```

Comment:
Looks like the code formatting changed here as well. Could be an issue with spotless.